shell/browser/file_system_access/file_system_access_permission_context.h

```diff
@@ -118,6 +118,11 @@ class FileSystemAccessPermissionContext
   bool OriginHasReadAccess(const url::Origin& origin);
   bool OriginHasWriteAccess(const url::Origin& origin);
 
+  // Called by FileSystemAccessWebContentsHelper when a top-level frame was
+  // navigated away from `origin` to some other origin. Is virtual for testing
+  // purposes.
```

Comment:
```suggestion
  // Is virtual for testing purposes.
```

:thinking:

Reply:
yes, this was modified from upstream and i just forgot to remove it!